src/common/types/store-data/user-configuration-store.ts

```diff
@@ -6,11 +6,11 @@ export interface UserConfigurationStoreData {
     enableHighContrast: boolean;
     issueTrackerPath?: string;
     bugService: string;
-    bugServicePropertiesMap: BugServicePropertiesMap;
+    bugServicePropertiesMap: IssueServicePropertiesMap;
```

Comment:
This is not being renamed for backward compatibility, yes?

Reply:
yes.